dlpy/applications/vgg.py

```diff
@@ -278,6 +278,8 @@ def VGG16(conn, model_table='VGG16', n_classes=1000, n_channels=3, width=224, he
     random_mutation : string, optional
         Specifies how to apply data augmentations/mutations to the data in the input layer.
         Valid Values: 'none', 'random'
+    reshape_after_input : Layer Reshape, optional
+        Specifies whether to add a reshape layer after the input layer.
```

Comment:
can we please also add a bit more comments here. something like explaining this if crucial for cnn/rnn models.

Reply:
It intends to be a general option not related to cnn/rnn.